spec/requests/feed_requests_spec.rb

```diff
@@ -36,6 +36,17 @@
       expect(first_entry["id"]).to eq(id)
       expect(first_entry["updated"]).to eq(updated)
     end
+
+    it "uses created_at instead of updated_at for updated value" do
+      created_at = 1.day.from_now
+      create(:redirection,
```

Comment:
Formatting:

```ruby
create(
  :redirection,
  created_at: created_at,
  updated_at: 2.days.from_now,
)
```

Also, how about defining `updated_at` in terms of `created_at` to show that they're different? Maybe `updated_at: created_at + 3.days`?